skcosmo/selection/CUR.py

```diff
@@ -0,0 +1,218 @@
+from abc import abstractmethod
+import numpy as np
+import scipy
+from scipy.sparse.linalg import eigs as speig
+
+from ._base import _BaseSelection
+
+class _CUR(_BaseSelection):
+    """
+    Super-class defined for CUR selection methods
+
+    Parameters
+    ----------
+    mixing : float
+        mixing parameter, as described in PCovR as `alpha`
+        stored in `self.mixing`
+    iterative: boolean
+        whether to orthogonalize the matrices after each selection
+    k : int
+        number of eigenvectors to compute the importance score with
+    matrix : ndarray of shape (n x m)
+        Data to select from -
+        Feature selection will choose a subset of the `m` columns
+        Samples selection will choose a subset of the `n` rows
+        s   tored in `self.A`
+    precompute : int
+        Number of selections to precompute
```

Comment:
that is not very clear. The docstring needs a high level overview of what is going on for users to be able to understand these parameters